test/platform-checks/mzcompose.py

```diff
@@ -99,18 +99,29 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
 
     args = parser.parse_args()
 
-    scenarios = (
-        [globals()[args.scenario]] if args.scenario else Scenario.__subclasses__()
-    )
-
-    checks = (
-        [globals()[check] for check in args.check]
-        if args.check
-        else Check.__subclasses__()
-    )
+    if args.scenario:
+        assert args.scenario in globals(), f"scenario {args.scenario} does not exist"
+        scenarios = [globals()[args.scenario]]
```

Comment:
Should we add an assertion here as well to make sure that `args.scenario` is of type `Scenario`?

Reply:
args.scenario is string here, but the check you want is on line  123 further down.

Reply:
Yes, I noticed 👍

Reply:
There was just a `for` loop existing already that iterated over the scenarios at line 122 so I reused it.